core/controllers/acl_decorators.py

```diff
@@ -145,8 +145,7 @@ def test_is_source_mailchimp(
         """
         if not email_manager.verify_mailchimp_secret(secret):
             logging.error(
-                'Invalid Mailchimp webhook request received with secret: %s'
-                % secret)
+                'Invalid Mailchimp webhook request received with secret')
```

Comment:
Maybe "incorrect secret"? Otherwise this doesn't really make sense.

Reply:
Done.